### DISCUSSION TITLE
功能建议：同步米家APP里的"手动控制“和”自动化“到HA

### DISCUSSION THREAD

**Discussion by xfengfeng** [XiaoMi/ha_xiaomi_home#203] [Ideas]
我之前已经在米家APP里添加了不少自动化规则，安装完HA登录帐号后，设备是导入了，但之前添加的自动化规则没有导入。
因部分人在传感器在HA里暂不支持，所以，还是需要在米家APP里添加部分关于人在传感器的自动化规则。

功能建议：
- 支持导入米家APP里的自动化规则到HA
- 自动化规则支持HA和米家APP的双向同步

Thanks.

**Comment by huhuime** [comment]
感觉上就不可能，连中枢极客版的都不在APP上显示，这个就更别想了
